luigi_pipeline/seqr_loading.py

```diff
@@ -1,16 +1,76 @@
+import logging
+
 import luigi
 
 from lib.hail_tasks import HailMatrixTableTask, HailElasticSearchTask, GCSorLocalTarget
+from lib.global_config import GlobalConfig
+
+logger = logging.getLogger(__name__)
 
 
+class SeqrValidationError(Exception):
+    pass
+
 class SeqrVCFToMTTask(HailMatrixTableTask):
     """
     Inherits from a Hail MT Class to get helper function logic. Main logic to do annotations here.
     """
     reference_mt_path = luigi.Parameter(description='Path to the matrix table storing the reference variants.')
+    sample_type = luigi.Parameter(description='Sample type, WGS or WES')
```

Comment:
If this is supposed to be either "WGS" or "WES", should it use [luigi.ChoiceParameter](https://luigi.readthedocs.io/en/stable/api/luigi.parameter.html#luigi.parameter.ChoiceParameter)?

Reply:
Nice, forgot about that when I was looking for ways to limit a Luigi Parameter.